config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  root 'root#index'
+
+  namespace :api do
+    namespace :v1 do
+      resources :greetings
```

Comment:
- [Optional] Using resourceful routes, 7 routes and their corresponding actions are generated by default which are not important for this specific. A single route and it corresponding action could have play out the trick perfectly :+1: